common/djangoapps/nodebb/tasks.py

```diff
@@ -88,6 +88,17 @@ def task_join_group_on_nodebb(category_id, username):
     handle_response(task_join_group_on_nodebb, 'Join user in category with id {}'.format(category_id), status_code, response, username)
 
 
+@task(default_retry_delay=RETRY_DELAY, max_retries=None)
+def task_un_join_group_on_nodebb(category_id, username):
+    """
+    Celery task to join user to a community on NodeBB
+    """
+    status_code, response = NodeBBClient().users.un_join(category_id=category_id, username=username)
+    handle_response(task_join_group_on_nodebb, 'Removed user from category with id {}'.format(category_id), status_code, response, username)
```

Comment:
Please check this part: should it be 'task_un_join_group_on_nodebb'?

Reply:
Nice catch. Actually, handle response was just to log the response that's why it passes the tests.